context/context.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
Yes, it is totally new. I intend that `context` package will include more functionalities besides `logger`, so I define a widely used name as `context`.